frontend/amundsen_application/static/js/pages/SearchPage/SearchFilter/InputFilter/index.tsx

```diff
@@ -146,21 +153,48 @@ export class InputFilter extends React.Component<
     updateFilterState(newFilters);
   };
 
-  render = () => {
-    const { value, filterOperation, showFilterOperationToggle } = this.state;
-    const { categoryId, allowableOperation } = this.props;
+  renderInputField = () => {
+    const { value } = this.state;
+    const { categoryId } = this.props;
     const inputAriaLabel = categoryId + 'FilterInput';
+    return (
+      <input
+        type="text"
+        className="form-control"
+        name={categoryId}
+        id={categoryId}
+        onChange={this.onInputChange}
+        value={value}
+        placeholder={INPUT_FILTER_PLACEHOLDER_MESSAGE}
+        aria-label={inputAriaLabel}
+      />
+    );
+  };
+
+  render = () => {
+    const { filterOperation, showFilterOperationToggle } = this.state;
+    const { categoryId, helpText, allowableOperation } = this.props;
+
+    const inputField = helpText ? (
+      <OverlayTrigger
+        trigger={['hover', 'focus']}
+        placement="top"
+        delayShow={DELAY_SHOW_POPOVER_MS}
+        overlay={
+          <Popover id={categoryId + '-help-text-popover'}>
+            <SanitizedHTML html={helpText} />
+          </Popover>
+        }
+      >
+        {this.renderInputField()}
+      </OverlayTrigger>
+    ) : (
+      this.renderInputField()
+    );
+
     return (
       <div>
```

Comment:
Now you can add a <></> instead of this dummy div (hopefully it will work the same)